internal/version/version.go

```diff
@@ -0,0 +1,4 @@
+package version
+
+// Version of the CLI
+const Version string = "0.1.0"
```

Comment:
Add the Release makefile / scripts to automatically update this ala https://github.com/newrelic/newrelic-client-go/blob/master/build/release.mk

Reply:
This felt out of scope for this PR, but I went ahead and did it.